src/main/java/com/jcabi/aether/Repository.java

```diff
@@ -99,8 +99,18 @@ public Repository(final RemoteRepository remote) {
         this.url = remote.getUrl();
         this.release = remote.getPolicy(false);
         this.snapshot = remote.getPolicy(true);
-        this.proxy = remote.getProxy();
-        this.authentication = remote.getAuthentication();
+        RepositoryAuthentication auth = null;
```

Comment:
How can modify this? the proxy and authentication are final and can be null in the remote object.